stream_interfaces.go

```diff
@@ -59,7 +101,21 @@ type ClientStreamingServer[Req any, Res any] interface {
 // request message stream and the type of the response message stream. It is
 // used in generated code.
 type BidiStreamingClient[Req any, Res any] interface {
+	// Send sends a message to the server. This method is called repeatedly
+	// to send all messages as part of the client’s request stream. The client
+	// can repeatedly call the Send method on the returned BidiStreamingClient
+	// stream in order to send the client-to-server message stream. If an error
+	// occurs on the stream, it will be returned as an instance of the 'status'
+	// package. Refer to the 'status' package documentation for more details.
 	Send(*Req) error
+
+	// Recv receives the next message from the server's response stream. This method
+	// is called repeatedly to receive all messages sent by the server. The client can
```

Comment:
```
This method
	// is called repeatedly to receive all messages sent by the server. The client can
	// repeatedly call Recv on this stream in order to receive the full server-to-client
	// message stream.
```

these sentences are looking talking about repeatedly twice. Let's combine in one.